velox/common/base/BitUtil.h

```diff
@@ -257,19 +257,19 @@ forEachWord(int32_t begin, int32_t end, PartialWordFunc partialWordFunc) {
 // outside of begin ... end by up to kWidth - 1 bits but using the
 // mask for example for load with mask will scope the operation to
 // valid values only.
-template <int8_t kWidth, typename Callable>
+template <uint8_t kWidth, typename Callable>
```

Comment:
This has no effect, compiler will just plug in the exact literal and there is no instruction generated.  Using signed number is safer in general.